Algorithm-Everything-be-True.md

```diff
@@ -1,6 +1,8 @@
+# Algorithm Everything be True

```

Comment:
Should this not be named `Algorithm Everything Be True` for both filename and title?